{{cookiecutter.project_name}}/Makefile

```diff
@@ -239,20 +239,35 @@ clean-all: clean clean-env .clean-workspace
 
 # Release ######################################################################
 
-.PHONY: register
-register: doc
-	$(PYTHON) setup.py register
-
 .PHONY: dist
 dist: check doc test tests
 	$(PYTHON) setup.py sdist
 	$(PYTHON) setup.py bdist_wheel
 	$(MAKE) read
 
+.PHONY: check-readme
+check-readme: doc
+	$(PYTHON) setup.py check -rsm
```

Comment:
What do you think about moving this line to somewhere in the `readme` target? I think `$ make readme` should fail if there's a problem with the contents.